webapp/channels/src/sass/routes/_get-app.scss

```diff
@@ -124,7 +124,7 @@
 }
 
 .get-app__preference {
-    display: flex;
+    display: block;
```

Comment:
I made this change to preserve the vertical alignment of the checkbox and the text, after removing the custom styling of the checkbox below.

Reply:
thank you its is a lot better let me add UX team for the review @abhijit-singh